compute_endpoint/globus_compute_endpoint/executors/high_throughput/interchange.py

```diff
@@ -555,6 +555,15 @@ def hold_manager(self, manager):
     def _status_report_loop(self, kill_event, status_report_queue: queue.Queue):
         log.info(f"Endpoint id: {self.endpoint_id}")
 
+        def _enqueue_status_report(ep_state: dict, task_states: dict):
+            try:
+                msg = EPStatusReport(self.endpoint_id, ep_state, dict(task_states))
```

Comment:
This is watching for RMQ errors, right?  If so, it would error out for all the chunks, so additional errors aren't necessarily useful.  Would catching the error outside the loop and NOT clearing the deltas and trying again in 30s be more productive?

I don't know the answer, dropping the deltas and logging may be fine.  But that's not just fixing the current 'bug' but changing the logic.

Reply:
Good points.  However, I'm inclined at the moment to leave it as is because nominally this whole section largely goes away once our unforking work goes in.